src/components/Library.tsx

```diff
@@ -1,49 +1,100 @@
-import React from 'react'
+import React, { useContext } from 'react'
 import { useTranslation } from 'react-i18next'
+import ContextProvider from '../state/ContextProvider'
 
 import { Game } from '../types'
 import GameCard from './UI/GameCard'
+import GameList from './UI/GameList'
 
 interface Props {
   library: Array<Game>
 }
 
+
+window.onscroll =  () => {
+  const pageOffset = document.documentElement.scrollTop || document.body.scrollTop,
+    btn = document.getElementById('backToTopBtn');
+  if (btn) btn.style.visibility = pageOffset > 450 ? 'visible' : 'hidden';
+};
+
+
 // TODO: Add a list options instead of Grid only
 export const Library = ({ library }: Props) => {
   const { t } = useTranslation()
+  const { layout } = useContext(ContextProvider)
+  const backToTop = () => {
+    const anchor = document.getElementById('top')
+    if (anchor) {
+      anchor.scrollIntoView({ behavior: 'smooth', block: 'center' });
+    }
+
+  }
 
   return (
-    <>
-      <div className="gameList">
-        {library.length ? (
-          library.map(
-            ({
-              title,
-              art_square,
-              is_dlc,
-              art_logo,
-              app_name,
-              isInstalled,
-            }: Game) => {
-              if (is_dlc) {
-                return null
+    <>     
+      {layout === 'grid' ? (
```

Comment:
Since the game list depends on this check, maybe we could use a ternary expression on the `className` and on the `return` 
Instead of repeating the whole code.